app/assets/javascripts/discourse/app/components/dialog-messages/confirm-session.gjs

```diff
@@ -119,12 +148,20 @@ export default class ConfirmSession extends Component {
               @label="user.password.confirm"
             />
           </div>
+          <div class="confirm-session__reset">
+            <a href {{on "click" this.sendPasswordResetEmail}}>
+              {{this.forgotPassword}}
+            </a>
+            <span class="confirm-session__reset-email-sent">
+              {{this.resetEmailSent}}
+            </span>
```

Comment:
I think we can avoid rendering this chunk of HTML unless `this.resetEmailSent` is not `null`

Reply:
Sure, updated.